jstl/src/main/java/com/sun/ts/tests/jstl/common/client/AbstractUrlClient.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2007, 2023 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2024 Oracle and/or its affiliates. All rights reserved.
```

Comment:
There are several instances where `2024` replaced `2007, 2023` or a similar date range. Shouldn't the second date just be updated? For example: `2007, 2024`.

Reply:
https://www.eclipse.org/projects/handbook/#ip-copyright-headers. The Eclipse practices seems to be changed to only require the initial year of file creation. I made the changes in latest commit.

Reply:
Ok interesting in the FAQ #2 it states:


```

Do we need to specify a range of years in the copyright statement?

No. In the past, legal advice was that the year of the initial creation of the content and the year of the last change should be reflected in the copyright header. This is no longer the case. Specify the year that the content was initially created in the copyright statement.
```

Reply:
I'll resolve this.